scripts/equadis-import/run_equadis_import.sh

```diff
@@ -12,11 +12,10 @@ find /home/sftp/equadis/data/ -mtime -2 -type f -exec cp {} /srv2/off-pro/equadi
 
 export NPM_CONFIG_PREFIX=~/.npm-global
 
-node equadis-xml2json.js
+node /srv/off-pro/scripts/convert_gs1_xml_to_json_in_dir.js /srv2/off-pro/equadis-data-tmp/
```

Comment:
why not use the new .pl here ?

Reply:
I want to test it out more thoroughly first, to check that we end up with the exact same results once the products are imported into OFF.